testsuite/features/core/proxy_register_as_minion_with_gui.feature

```diff
@@ -12,6 +12,10 @@ Feature: Setup SUSE Manager proxy
   As the system administrator
   I want to register the proxy to the server
 
+@proxy
+  Scenario: Install proxy pattern on the proxy
+    When I install pattern "suma_proxy" on this "proxy"
+
```

Comment:
:thinking: I'm curious now to understand why we don't need in CI the step `And I set correct product for "proxy"`

Reply:
Yes, I need to ask to Lukas.

I'm not even sure to understand this workaround, to be honest.

Reply:
We decided to just use it everywhere